src/raftstore/store/fsm/store.rs

```diff
@@ -423,8 +421,15 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             self.cfg.use_delete_range,
             self.cfg.clean_stale_peer_delay.0,
         );
-        let mut timer = Timer::new(1);
-        timer.add_task(Duration::from_millis(STALE_PEER_CHECK_INTERVAL), ());
+        let mut timer = Timer::new(2);
```

Comment:
How about wrapping it in `region_worker`'s method?

Reply:
No, `region_worker` is `Worker` and other `xxx_worker`s use `Worker` too, so can not wrap these specific timers into `Worker` logic.

Reply:
Here is what I mean https://github.com/tikv/tikv/blob/e1b04eba6c0a2851fb4aab798bec2a06c3ae1ad4/src/raftstore/store/worker/read.rs#L288

Reply:
yeah, it is what I mean. Is add a `new_timer()` for Worker that returns these two specific timers, it would be weird for other workers

Reply:
I mean add it to runner not worker.